stubs/inertia/resources/js/Pages/Auth/Register.vue

```diff
@@ -79,18 +79,15 @@
                     email: '',
                     password: '',
                     password_confirmation: '',
-                    terms: null,
+                    terms: false,
                 })
             }
         },
 
         methods: {
             submit() {
                 this.form.post(this.route('register'), {
-                    onSuccess: () => {
-                        this.form.password = ''
-                        this.form.password_confirmation = ''
-                    }
+                    onFinish: () => this.form.reset('password', 'password_confirmation'),
```

Comment:
Curious, why do we need to reset these fields at all? Doesn't the user get redirected after they register? 🤔

Reply:
Primarily the reasoning here is that if there's errors on the page, you do want to reset the passwords fields.
Furthermore, it's never a bad idea to clear this kind of sensitive data when you can, as you don't want to accidentally retain them in history/storage.